.drone.yml

```diff
@@ -1,75 +1,106 @@
+---
+kind: pipeline
+name: testing
+
+platform:
+  os: linux
+  arch: amd64
+
 workspace:
   base: /go
   path: src/code.gitea.io/gitea
 
-pipeline:
-  fetch-tags:
-    image: docker:git
-    commands:
-      - git fetch --tags --force
+services:
+  - name: mysql
+    pull: default
+    image: mysql:5.7
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
+      MYSQL_DATABASE: test
     when:
       event:
-        exclude: [ pull_request ]
+        - push
+        - tag
+        - pull_request
 
-  download_translations:
-    image: jonasfranz/crowdin
-    pull: true
-    secrets: [ crowdin_key ]
-    project_identifier: gitea
-    ignore_branch: true
-    download: true
-    export_dir: options/locale/
+  - name: mysql8
+    pull: default
+    image: mysql:8.0
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
+      MYSQL_DATABASE: testgitea
     when:
-      event: [ push ]
-      branch: [ master ]
+      event:
+        - push
+        - tag
+        - pull_request
 
-  update-translations:
-    image: alpine:3.10
-    commands:
-      - mv ./options/locale/locale_en-US.ini ./options/
-      - sed -i -e 's/="/=/g' -e 's/"$$//g' ./options/locale/*.ini
-      - sed -i -e 's/\\\\"/"/g' ./options/locale/*.ini
-      - mv ./options/locale_en-US.ini ./options/locale/
+  - name: pgsql
+    pull: default
+    image: postgres:9.5
+    environment:
+      POSTGRES_DB: test
     when:
-      event: [ push ]
-      branch: [ master ]
+      event:
+        - push
+        - tag
+        - pull_request
 
-  git_push:
-    image: appleboy/drone-git-push
-    pull: true
-    secrets: [ git_push_ssh_key ]
-    remote: git@github.com:go-gitea/gitea.git
-    force: false
-    commit: true
-    commit_message: "[skip ci] Updated translations via Crowdin"
-    author_name: GiteaBot
-    author_email: teabot@gitea.io
+  - name: mssql
+    pull: default
+    image: microsoft/mssql-server-linux:latest
+    environment:
+      ACCEPT_EULA: Y
+      MSSQL_PID: Standard
+      SA_PASSWORD: MwantsaSecurePassword1
+    when:
+      event:
+        - push
+        - tag
+        - pull_request
+
+  - name: ldap
+    pull: default
+    image: gitea/test-openldap:latest
     when:
-      event: [ push ]
-      branch: [ master ]
+      event:
+        - push
+        - tag
+        - pull_request
 
-  pre-build:
+steps:
+  - name: fetch-tags
+    pull: default
+    image: docker:git
+    commands:
+      - git fetch --tags --force
```

Comment:
Can you exclude this step from running on PRs?

Reply:
Done.